README.md

```diff
@@ -4,27 +4,242 @@
+
+
+## Pre-requisites
+
+1. Make sure you have Java 1.7 to 1.8 installed
+2. Make sure you have Android Studio installed or [download it from here](https://developer.android.com/studio/index.html)
+
+
+## Installation Devices
+
+1. Use a physical Android device to run the app
+2. Use the Android Emulator that comes with the Android Studio installation (Slow & not advisable)
+3. Use Genymotion Android Emulator
+    * Go [here](https://www.genymotion.com/) and register for genymotion account if none. Free accounts have limitations which are not counter-productive
+    * Download your OS Version of VirtualBox at [here](https://www.virtualbox.org/wiki/Downloads)
+    * Install VirtualBox
+    * Download Genymotion & Install it
+    * Sign in to the genymotion app
+    * Create a new Genymotion Virtual Device 
+        * **Preferrable & Stable Choice** - API 22(Android 5.1.0), Screen size of around 800 X 1280, 1024 MB Memory --> eg. Google Nexus 7, Google Nexus 5
+
+## How to install
+
+1. Import the project into Android Studio by: **Import a gradle project** option
+   _All the plugins required are explicitly stated, therefore it can work with any Android Studio version - Just enable it to download any packages not available offline_
+1. Open Genymotion and Run the Virtual Device created previously.
+1. Run the app on Android Studio and chose the Genymotion Emulator as the ` Deployment Target`
+
+
+### Developer Guides
+
+If you want to contribute please refer to these resources:
+
+   * [Getting started with OpenSRP](https://smartregister.atlassian.net/wiki/spaces/Documentation/pages/6619148/Getting+started+with+OpenSRP)
+   * [Setup Instructions](https://smartregister.atlassian.net/wiki/spaces/Documentation/pages/6619255/Setup+Instructions)
+   * [Complete OpenSRP Developer's Guide](https://smartregister.atlassian.net/wiki/spaces/Documentation/pages/6619193/OpenSRP+Developer%27s+Guide)
+
+### Wiki
+
+If you are looking for detailed guides on how to install, configure, contribute and extend OpenSRP visit [OpenSRP Wiki](https://smartregister.atlassian.net/wiki)
+
+
+# Uses
+
+OpenSRP Client core has been used in several libraries and applications:
+
+   * [OpenSRP Path application](https://github.com/OpenSRP/opensrp-client-path)
+   * [OpenSRP KIP application](https://github.com/OpenSRP/opensrp-client-kip)
+   * [OpenSRP Growth monitoring library](https://github.com/OpenSRP/opensrp-client-growth-monitoring)
+   * [OpenSRP Immunization library](https://github.com/OpenSRP/opensrp-client-immunization)
+   * [OpenSRP Native Form library](https://github.com/OpenSRP/opensrp-client-native-form)
+
+
+# Main Functions
+
+## 1. Security
+
+Security is provided in the following:
+   * Network - It supports ssl in cases where the backend server is connected through a `https` url
+   * Data access - Only registered providers can be able to view and manipulate records
+   * Data encryption - The database on the android client is encrypted
```

Comment:
let's state the protocol. What's the default encryption, is it https://guardianproject.info/code/sqlcipher/? If so, let's add that to the end, like "encrypted with 256-bit AES encryption using [SQLCipher](https://guardianproject.info/code/sqlcipher/)."